packages/react-native/src/components/QuilttConnector.tsx

```diff
@@ -84,7 +84,7 @@ export const handleOAuthUrl = (oauthUrl: string | URL) => {
     // Open the normalized URL
     Linking.openURL(normalizedUrl)
   } catch (error) {
-    // @todo: Report to HB
+    errorReporter.notify(new Error('OAuth URL handling error'), { oauthUrl, error })
```

Comment:
My concern here here is that errorReporter is not actually enabled for most clients because we're not shipping our API key. 

So i think this will fail silently yet again.